magento/pwa-studio
# [chore]: Fixing Prettier use

## Description

1) Removes unnecessary `prettier-check` dependency and modifies pre-push to use `prettier:check` instead of `prettier:validate`.

2) Creates a `prettier.config.js` file and uses the config version of `--single-quote`. I think this is a better experience for developers for future configuration on top of venia-concept.

## Related Issue





Closes [PWA-89](https://jira.corp.magento.com/browse/PWA-89).

## Acceptance 



### Verification Stakeholders
@zetlen - I'd like to know if there are any additional changes necessary in `venia-concept` or in the scaffolding scripts.

### Specification



### Verification Steps


1. Make some change in `venia-concept` that would fail prettier like adding double quotes to an import. Make sure you have "format on save" off so that it doesn't auto fix. 
2. Run `yarn prettier:check` from the root directory. It should complain.
3. Go to `packages/venia-concept` and run `yarn prettier`. It should also complain.

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
